board/can.h

```diff
@@ -21,7 +22,7 @@ void can_init(CAN_TypeDef *CAN, int silent) {
     CAN->BTR |= CAN_BTR_SILM | CAN_BTR_LBKM;
   #endif
 
-  if (silent) {
+  if (!controls_allowed) {
```

Comment:
I will split silent and controls allowed. Should silent have a way to be re-enable after the "controls allowed" usb command is called?